master/usage/decommissioning-a-node.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Decommissioning a Calico Node
+---
+
+With each Calico node there is a [Node resource][Node resource reference]
```

Comment:
Yeah this first para isn't snappy enough.

Can't we just say something like:  

If you are decommissioning a node (e.g. because it is being taken out of service, or it is being renamed), it is necessary to explicitly delete the associated Node resource to remove any data associated with that node.

In environments where nodes will be added ...